MAUI/ListView/item-drag-and-drop.md

```diff
@@ -0,0 +1,485 @@
+---
+layout: post
+title: Item Reordering in .NET MAUI ListView control | Syncfusion
```

Comment:
title: Item Reordering in .NET MAUI ListView Control | Syncfusion

Reply:
Addressed the concerns provided.